partiql-eval/src/eval.rs

```diff
@@ -880,6 +923,7 @@ impl EvalExpr for EvalUnaryOpExpr {
     }
 }
 
+/// Represents a PartiQL evaluation `IS` operator, e.g. `IS` in `a IS INT`.
```

Comment:
Think this technically models the whole expression `a IS INT`, not just the `IS` part